acra/acra-in-depth/security-design/_index.md

```diff
@@ -2,3 +2,165 @@
 title: Security design
 bookCollapseSection: true
 ---
+
+## Security design
+
+Acra is built to provide selective encryption only to the records that require such protection.
+AcraWriter provides the app developer with the means to encrypt these records via wrapping anything in a function that outputs [AcraStruct]({{< ref "acra/acra-in-depth/data-structures/#understanding-acrastruct" >}}), a crypto container decryptable by AcraServer, which is then stored in a database.
+
+Any secret contained within an app (i.e. passwords in config files) can be leaked. There are many reasons why a developer might not want to use the user password as a secret - not all systems have to be end-to-end encrypted, even though we'd like it to be that way. An example of a good trust model is to use asymmetric cryptography to "send" this data (for the encryption of which we only have a public key) to some remote party. A leakage of this public key will still be insufficient for decrypting anything from the database.
```

Comment:
> **i.e.** passwords in config files

This should be “e.g.”

i.e. = *id est*, “that is”, used to explain or rephrase something
e.g. = *exempli gratia*, “for example”, used to introduce examples